tests/test_gui.py

```diff
@@ -515,10 +515,19 @@ def test_note_editor(app_dialog):
     ].exists(), "Not on the Activity Stream widget"
     assert app_dialog.root.captions["New Note"].exists(), "New Note is missing"
     # Get the current user
+    sg = get_toolkit_user().create_sg_connection()
     user = str(get_toolkit_user())
-    assert app_dialog.root.captions[
-        user + "*"
-    ].exists(), "Not the right user linked to the note"
+    # Validate if it is an API or human user
+    if sg.find_one("ApiUser", [["firstname", "is", user]]) is None:
```

Comment:
A better way to find the human or script user is to do this:
```
user = get_toolkit_user()
if isintance(user, sgtk.authentication.ScriptUser):
   ...
else:
   ...
```

Reply:
isinstance(user, sgtk.authentication.ScriptUser) is not working so instead I'm using your other suggestion:
```
if user.login is None:
    # You have a script user
    ...
else:
    # You have a human user
    ...
```